libraries/AP_Notify/RGBLed.cpp

```diff
@@ -262,7 +262,7 @@ void RGBLed::update_colours(void)
             // fast flashing green if disarmed with GPS 3D lock and DGPS
             // slow flashing green if disarmed with GPS 3d lock (and no DGPS)
             // flashing blue if disarmed with no gps lock or gps pre_arm checks have failed
-            bool fast_green = AP_Notify::flags.gps_status >= AP_GPS::GPS_OK_FIX_3D_DGPS && AP_Notify::flags.pre_arm_gps_check;
+            bool fast_green = AP_Notify::flags.gps_status > AP_GPS::GPS_OK_FIX_3D_DGPS && AP_Notify::flags.pre_arm_gps_check;
```

Comment:
I think it should be left as before. its handy for trouble shooting with Multi. The user however can just be instructed to look at GPS status from a GUI or something. Anyone using the multi will most likely be flying with some GCS.